HighwayPatrol.h

```diff
@@ -1,11 +1,22 @@
 #pragma once 
 
 #include "Vehicle.h"
+#include <string>
+
+struct Vehicle;
```

Comment:
don't forward declare something if you're including the header file for it
- [ ] revise